addons/mail/static/src/core/web/composer_patch.js

```diff
@@ -5,12 +5,152 @@ import { childNodes } from "@html_editor/utils/dom_traversal";
 import { Composer } from "@mail/core/common/composer";
 import { createDocumentFragmentFromContent } from "@mail/utils/common/html";
 
-import { markup } from "@odoo/owl";
+import { markup, toRaw, useEffect } from "@odoo/owl";
 
 import { patch } from "@web/core/utils/patch";
 import { renderToElement } from "@web/core/utils/render";
+import { fixInvalidHTML } from "@html_editor/utils/sanitize";
+import { MAIL_PLUGINS, MAIL_SMALL_UI_PLUGINS } from "./plugins/plugin_sets";
+import { Wysiwyg } from "@html_editor/wysiwyg";
+import { useService } from "@web/core/utils/hooks";
+import { isEmptyHtmlBody } from "@mail/utils/common/format";
+
+patch(Composer, {
+    components: { ...Composer.components, Wysiwyg },
+});
 
 patch(Composer.prototype, {
+    setup() {
+        super.setup();
+        this.wysiwyg = {
+            config: this.wysiwygConfigs,
+            editor: undefined,
+        };
+        this.suggestion = null;
+        useEffect(
+            (focus) => {
+                if (focus && this.wysiwyg.editor) {
+                    this.wysiwyg.editor.shared.selection.setCursorEnd(
+                        this.wysiwyg.editor.editable.lastChild
+                    );
+                    this.wysiwyg.editor.shared.selection.focusEditable();
+                    this.wysiwyg.editor.editable.focus();
+                }
+            },
+            () => [this.props.autofocus + this.props.composer.autofocus, this.props.placeholder]
+        );
+    },
+    get wysiwygConfigs() {
+        return {
+            content: fixInvalidHTML(this.props.composer.htmlBody) || "<p><br></p>",
```

Comment:
markup should be used, I don't know (yet) how `content` is used, but the code making use of it should not be working without content being markup if it is safely built